zerver/models.py

```diff
@@ -1870,6 +1871,12 @@ def has_permission(self, policy_name: str) -> bool:
         if policy_value == Realm.POLICY_EVERYONE:
             return True
 
+        if self.is_realm_owner:
+            return True
+
+        if policy_value == Realm.POLICY_OWNERS_ONLY:
+            return False
```

Comment:
This is new security-sensitive code, and should have test coverage in the main test for `has_permission`; I'm a bit surprised that this commit passes tests.

Reply:
Added tests for `Owners only case and also added the tests for `Everyone`, `Nobody` as prep commits.

Reply:
These are great, thanks!